src/components/Calculator.js

```diff
@@ -1,30 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import calculate from '../logic/calculate';
 import './calculator.css';
 
-const Calculator = () => (
-  <>
-    <section className="Calculator-Container">
-      <div className="signs result">0</div>
-      <div className="signs">AC</div>
-      <div className="signs">+/-</div>
-      <div className="signs ">%</div>
-      <div className="signs rose">÷</div>
-      <div className="signs">7</div>
-      <div className="signs">8</div>
-      <div className="signs">9</div>
-      <div className="signs rose">x</div>
-      <div className="signs">4</div>
-      <div className="signs">5</div>
-      <div className="signs">6</div>
-      <div className="signs rose">-</div>
-      <div className="signs">1</div>
-      <div className="signs">2</div>
-      <div className="signs">3</div>
-      <div className="signs rose">+</div>
-      <div className="signs zero">0</div>
-      <div className="signs rose">=</div>
-    </section>
-  </>
-);
-
-export default Calculator;
+const Calculator = ({ data, result }) => {
+  const handleClick = (event) => {
+    data(event);
+  };
+
+  return (
+
+    <>
+      <section className="Calculator-Container">
+        <div className="signs result">{result}</div>
+        <button name="AC" type="button" onClick={handleClick} className="signs">AC</button>
+        <button name="+" type="button" onClick={handleClick} className="signs">+/-</button>
```

Comment:
- [ ] As required for the project, kindly make sure all buttons work as expected. Right now, I noticed that you made an error on the `+/-` button. You assign the `name` attribute to `+` instead of `+/-` as result it works as a `+` operator. Ensure that you fix it to match the required expectations.